modules/tile-converter/src/i3s-server/controllers/index-controller.ts

```diff
@@ -12,9 +12,16 @@ const I3S_LAYER_PATH = process.env.I3sLayerPath || '';
  */
 export async function getFileNameByUrl(url: string): Promise<string | null> {
   const extensions = ['json', 'bin', 'jpg', 'jpeg', 'png', 'bin.dds', 'ktx2'];
-  const FULL_LAYER_PATH = path.join(process.cwd(), I3S_LAYER_PATH);
+  const absolutePath = /^[^.]+/.exec(I3S_LAYER_PATH);
```

Comment:
Add a comment for regexp. It always anoying to read them.

Reply:
Removed regex